src/lib/config.js

```diff
@@ -32,9 +47,66 @@ async function config() {
       console.error(err);
     }
     log(chalk.red(str));
-    return;
+    process.exit(1);
   }
+}
 
+async function createLightnetDeployAlias(projectRoot, deployAliasesConfig) {
+  const networkId = 'testnet';
+  const deployAliasPrefix = 'lightnet';
+  const minaGraphQlEndpoint = 'http://localhost:8080/graphql';
+  let nextAliasNumber = 1;
+  await step(`Check Mina GraphQL endpoint availability`, async () => {
```

Comment:
This means that you _need_ lightnet running to generate a config? I'm not sure how I feel about that :/ Can you eloborate more on why that should be the case?

Reply:
Because if we want it to work without any additional activities we need to retrieve accounts (fee payer) information from actual lightnet. Otherwise we will end up duplicating ledger configuration here within the zkapp-CLI repo and the lightnet itself, or some other additional and not really convenient steps that might negate this improvement.
Unless I'm missing something here. Maybe you were thinking about some other options?

Reply:
I was just wondering the reasoning and if we could avoid it. That's a good point about fetching the fee payer! I didn't think about that case, makes sense!